sunscreen_runtime/src/debugger/data.rs

```diff
@@ -305,11 +305,23 @@ fn create_plaintext_from_seal_data(pt: SealPlaintext, pk: &PrivateKey) -> InnerP
     sunscreen_plaintext.inner
 }
 
-#[derive(Clone, Serialize, Deserialize)]
+#[derive(Clone, Deserialize)]
 pub struct ZkpNodeType {
     // Send `BigInt` values as strings
     pub value: String,
 }
 
+impl Serialize for ZkpNodeType {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        match self.value.find(|c: char| c.is_numeric() && c != '0') {
+            Some(_) => serializer.serialize_str(self.value.trim_start_matches('0')),
```

Comment:
There shouldn't be a problem parsing leading zeros on the frontend? This type seems kind of unnecessary but probably ok to leave for now

Reply:
There's a excessive amount of leading zeroes. I was unsure if that was something that would be better to tackle on the frontend or backend.

Reply:
It is a `512` bit number, so there are 64 bytes. But as long as its a string, you can pass it to the JS BigInt constructor:
```js
BigInt('0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a')
# 10n
```
If you want, you could just `#[derive(Serialize)]` on the BigInt, and also specify `#[serde(transparent)]` to use the underlying U512 serialization, but that will be a little endian hex encoding and requiring handling on the frontend.

I think you can keep this as is for now.